crates/server/src/store/blocklist/mod.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-2022 Benjamin Fry <benjaminfry@me.com>
```

Comment:
The `mod.rs`/`config.rs`/`authority.rs` split results in pretty small modules. I don't recall off the top of my head if that style is prevalent here, but I would probably keep everything in a single `mod.rs` for now (or maybe move the `Config` into the `mod.rs`).

Reply:
> The `mod.rs`/`config.rs`/`authority.rs` split results in pretty small modules. I don't recall off the top of my head if that style is prevalent here, 

It is - I copied the structure of the other authority implementations.